lib/avo/filters/base_filter.rb

```diff
@@ -66,6 +66,7 @@ def applied_or_default_value(applied_filters)
       # Fetch the applied filters from the params
       def applied_filters
         return {} if (filters_from_params = params[PARAM_KEY]).blank?
+        return {} if !filters_from_params.is_a?(String)
```

Comment:
Let's add a comment on why this is here.
And make this the default pattern from now on.